config/ksql-production-server.properties

```diff
@@ -74,9 +74,11 @@ ksql.streams.producer.delivery.timeout.ms=2147483647
 # Kafka cluster is unavailable.
 ksql.streams.producer.max.block.ms=9223372036854775807
 
-# For better fault tolerance and durability, set the replication factor for the KSQL
-# Server's internal topics. Note: the value 3 requires at least 3 brokers in your Kafka cluster.
+# For better fault tolerance and durability, set the replication factor and minimum insync replicas
```

Comment:
If you're up for it, I think we should split all of the new configuration work out into a separate PR and get that one merged quickly - you'll be able to make some progress on that front and it'll also make this one easier to review.

Reply:
I think it's not worth the effort to break it out. The configuration for the command topic effects 4 files, which I don't think adds too much complexity to a review.

```
ksql-production-server.properties
KsqlConfig
KsqlInternalTopicUtil and test file
```

Reply:
just a thought for untying these - but if there's no rush then I don't mind :)